client/components/sites-dropdown/index.jsx

```diff
@@ -104,5 +114,6 @@ export class SitesDropdown extends PureComponent {
 export default connect(
 	( state ) => ( {
 		primarySiteId: getPrimarySiteId( state ),
+		siteCount: get( getCurrentUser( state ), 'site_count', 0 )
```

Comment:
As we discussed, it would be nice to do a simple refactoring and pass down to the component `hasMultipleSites` boolean flag to avoid several code calls like `this.props.siteCount > 1`.